starknet_devnet/blueprints/rpc/transactions.py

```diff
@@ -245,12 +245,19 @@ async def simulate_transaction(
             skip_validate=skip_validate,
             block_id=block_id,
         )
-        simulated_transactions.append(
-            {
-                "transaction_trace": rpc_map_traces(traces, transaction_types),
-                "fee_estimation": rpc_fee_estimate(fee),
-            }
-        )
+        rpc_traces = rpc_map_traces(traces, transaction_types)
+        rpc_estimations = rpc_fee_estimate(fee)
+
+        # traces number must be equal to estimations
+        assert len(rpc_traces) == len(rpc_estimations)
+        for i in range(len(rpc_traces)):
```

Comment:
The pythonic way is to do
```python
for trace, estimation in zip(rpc_traces, rpc_estimations):
    ...
```

Reply:
fixed in b1424deac32fb75351bbbca5fbc0418556a23bf9